--- a/src/st_stuff.c	2026-05-02 09:58:06.130820022 +0000
+++ b/src/st_stuff.c	2026-05-02 09:58:33.197269961 +0000
@@ -248,9 +248,6 @@
 // ST_Start() has just been called
 static dbool st_firsttime;
 
-// used to execute ST_Init() only once
-static int veryfirsttime = 1;
-
 // CPhipps - no longer do direct PLAYPAL handling here
 
 // used for timing
@@ -1188,6 +1185,5 @@
 */
 void ST_Init(void)
 {
-  veryfirsttime = 0;
   ST_loadData();
 }
